exercises/outcomes/LT/LT7/generator.sage

```diff
@@ -0,0 +1,79 @@
+class Generator(BaseGenerator):
+    def data(self):
+        var_letter = "x" # choice(["x", "y", "h"])
+        x = var(var_letter)
+        problem_kind = randrange(1, 5)
+        # pk = 1: sin nx / nx ; pk = 2: sin nx / x ; pk = 3: cos nx - 1 / x ; pk = 4 : sin(nx)^2 / (nx)^2
+        if problem_kind == 1:
+            n = randrange(2, 10)
+            f = sin(n*x) / (n*x)
+            lim = limit(f, x=0)
```

Comment:
This works but is probably what's making generation so slow. Can't we just say `lim = 1` here, because math?

Reply:
In the final draft, naturally 🤣